plugins/modules/django_check.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: django_check
+author:
+  - Alexei Znamensky (@russoz)
+short_description: Wrapper for C(django-admin check)
+version_added: 9.1.0
+description:
+  - This module is a wrapper for the execution of C(django-admin check).
```

Comment:
How does the module reports the result? Does it fail if the check fails? Maybe add a sentence in the description about that.

Reply:
I will write a bit more, but basically it reports with raw stdout, stderr, rc - no parsing so far. I am happy to add parsing later, right now I am trying to get a couple of  django modules out.

Reply:
Failing (due to rc check) is fine for me, I think it just should be documented :) And parsing can always be added later.

Reply:
@felixfontein done!